TricksterBots/Bots/Euchre/EuchreBot.cs

```diff
@@ -77,6 +84,7 @@ public override BidBase SuggestBid(SuggestBidState<EuchreOptions> state)
 
             var isDealer = player.Seat == dealerSeat;
             var isTeamDealing = isDealer || players.PartnersOf(player).Any(p => p.Seat == dealerSeat);
+            var willLeadFirst = (state.dealerSeat + 1) % state.options.players == state.player.Seat;
```

Comment:
Need to check later if we're making an alone bid in NT and "alone leads left" is enabled.